geowebcache/diskquota/jdbc/src/main/java/org/geowebcache/diskquota/jdbc/JDBCQuotaStore.java

```diff
@@ -203,10 +203,12 @@ public String mapRow(ResultSet rs, int rowNum) throws SQLException {
                     deleteLayer(layerName);
                 }
 
+                // The tileset is created by #addHitsAndSetAccesTime when used
+
                 // add any missing tileset
-                for (String layerName : layerNames) {
-                    createLayerInternal(layerName);
-                }
+                // for (String layerName : layerNames) {
+                // createLayerInternal(layerName);
+                // }
```

Comment:
Again, comment explaining where this is being done elsewhere is good, commented out code is bad.